base/windows/shop/EquipCompare.js

```diff
@@ -198,15 +235,16 @@ export class EquipCompare {
                 break;
         }
 
-        let atk_diff = this.compare_items(char_current_item, this.selected_item, "attack");
-        let def_diff = this.compare_items(char_current_item, this.selected_item, "defense");
-        let agi_diff = this.compare_items(char_current_item, this.selected_item, "agility");
+        let atk_diff = this.compare_items(char_current_item, this.selected_item, effect_types.ATTACK, this.selected_char.current_atk);
+        let def_diff = this.compare_items(char_current_item, this.selected_item, effect_types.DEFENSE, this.selected_char.current_def);
+        let agi_diff = this.compare_items(char_current_item, this.selected_item, effect_types.AGILITY, this.selected_char.current_agi);
```

Comment:
please use atk, def and agi instead of current_atk, current_def and current_agi. I'm planning to remove those in the future.